setup.py

```diff
@@ -14,9 +14,7 @@
     author_email="emmanuel.leblond@gmail.com",
     license="MIT -or- Apache License 2.0",
     packages=find_packages(),
-    install_requires=[
-        "trio",
-    ],
+    install_requires=["trio", "hypothesis==4.4.3"],
```

Comment:
Generally speaking, the recommendatiom is that installable libraries like this should *not* do exact pins. Why is this exact version of hypothesis needed?

Reply:
The problem here is that `hypothesis_trio/stateful.py` is mostly a copy-paste from functions and classes from `hypothesis/stateful.py` with a bit of extra trio stuff. So there's a very good chance that it's not compatible with previous or future versions of hypothesis. I can check that in the file history.

A better approach would probably be to add support for async state machines to hypothesis. I guess this should be discussed  with the maintainers of the project.